packages/asc-ui/src/index.ts

```diff
@@ -75,6 +75,7 @@ import {
 import Modal from './components/Modal'
 import Paragraph, { ParagraphStyle } from './components/Paragraph'
 import Portal from './components/Portal'
+import Quote, { QuoteStyle } from './components/Quote'
```

Comment:
Perhaps we can use a `export { default as Quote, QuoteStyle }` statement here?

Reply:
gedaan